src/App.jsx

```diff
@@ -1,39 +1,39 @@
 import './App.css'
 import {BrowserRouter, Routes, Route} from 'react-router-dom';
+import {apiKey, baseId, tableName} from '../config.js';
 import TodoContainer from './components/TodoContainer.jsx'
 import NavBar from './components/NavBar.jsx';
 import HomePage from './pages/HomePage';
-import {apiKey, baseId, tableName} from '../config.js';
-import listIcon from "./img/list.png";
-import style from "./components/TodoContainer.module.css";
+import NewListPage from './pages/NewListPage';
+import NotFoundPage from './pages/NotFoundPage';
 import TodoListSelector from './components/TodoListSelector';
+import Footer from './components/Footer';
 
 
 function App() {
     return (
         <BrowserRouter>
             <NavBar/>
-            <Routes>
-                <Route exact path="/" element={<HomePage />} />
-                <Route path="/todos/:listName" element={
-                    < TodoContainer
-                        tableName={tableName}
-                        baseId={baseId}
-                        apiKey={apiKey}/>
-                } />
-                <Route exact path="/mylists" element={
-                    <TodoListSelector
+            <main>
+                <Routes>
+                    <Route exact path="/" element={<HomePage/>}/>
+                    <Route path="*" element={<NotFoundPage/>}/>
+                    <Route path="/todos/:listName" element={
```

Comment:
it is very interesting behavior here. If I go to the route "/todos/:listName" and print not existing name for the list, the app instead of showing me `NotFoundPage` allow to create new list with the name I entered to check NotFoundPage functionality

Reply:
This is because the variable 'listName' is used across functions for navigating to an existing list, and for creating a new one.

Reply:
I see, just vant to be sure you are aware of this behavior

Reply:
":" is not shown in any list end point, and any non-existing end point like /todos/anyname leads to the 404 page. 
SO /todos/:listName (with semicolon) is only use for creating a new list, and common user usually does not put ":" in end point name.